### PR TITLE
MP Rest Client 1.2

### LINKED ISSUE #43: Rest client 1.2-m1
I am working on OpenTracing instrumentation for rest-client 1.2. To verify the integration I would like to run TCK but I need rest-client implementation based on 1.2-m1 https://github.com/eclipse/microprofile-rest-client/releases/tag/1.2-m1.

Is anybody working on the implementation? When it's going to be done?

### PR DESCRIPTION
**Pull request by michalszynkiewicz**
fixes #43

### CODE REVIEW

**Comment by michalszynkiewicz** [comment]
Thanks Ken for switching to the released version!

I tried the tests locally and they pass. I also retried them a couple times on SemaphoreCI and they fail (always on time out on wiremock but not always the same amount of tests fails...)

**Comment by kenfinnigan** [comment]
I suspected it was something intermittent.

So I guess we can take a punt and merge and hope it works?

**Comment by michalszynkiewicz** [comment]
@kenfinnigan if you're okay with it we can do it ;)

**Comment by kenfinnigan** [comment]
If it's run locally without issue we can give it a shot.

Then investigate further if it fails again, maybe we need to increase a wiremock timeout

File: implementation/src/main/java/io/smallrye/restclient/MethodInjectionFilter.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2015-2017 Red Hat, Inc, and individual contributors.
```
Comment (kenfinnigan) [inline implementation/src/main/java/io/smallrye/restclient/MethodInjectionFilter.java]: Should this be 2019?

**Pull request merged**